tuftool/src/create.rs

```diff
@@ -51,6 +52,12 @@ pub(crate) struct CreateArgs {
     #[structopt(short = "t", long = "add-targets")]
     targets_indir: PathBuf,
 
+    /// Behavior when a target exists with the same name and hash in the desired repository
+    /// directory, for example from another repository when you're sharing target directories.
+    /// Options are "replace", "fail", and "skip"
```

Comment:
Yeah, this could be clarified.  I thought it was important to give an example because it's not immediately apparent to everyone how you could have matching target filenames/hashes, and maybe not everyone is sharing a targets directory across repos.  (That's the only case we could think of.)  How about:

```
     /// Behavior when a target exists with the same name and hash in the targets
     /// directory, for example from another repository when they share a targets directory.
    /// Options are "replace", "fail", and "skip".
```